csrc/python_frontend/fusion_record.h

```diff
@@ -2443,6 +2721,21 @@ struct RandomOpRecord : RecordFunctor {
     }
   }
 
+  virtual std::pair<serde::RecordData, flatbuffers::Offset<void>> recordData(
+      flatbuffers::FlatBufferBuilder& builder) const final {
+    std::vector<serde::State> fb_shape;
+    for (auto& it : output_shape_) {
+      fb_shape.push_back(serde::State(it.index, it.stype));
+    }
+    auto shape_fb =
```

Comment:
This variable is not used, is this a bug?

Reply:
It isn't need. Not sure why it didn't show up in clang-tidy.

Reply:
I removed it here. https://github.com/NVIDIA/Fuser/pull/114